.github/workflows/fastapi.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     defaults:
       run:
         working-directory: fastapi
+
```

Comment:
looks like you still have changes to this file.

you can just revert entirely with `git checkout origin/master -- .github/workflows/fastapi.yml` to get the copy of the file that exists on `master` (still need to `git add/commit` the revert, but that is an easy trick to restore to the same file as it is on `master`)